consoles/sshVirtsh.pm

```diff
@@ -320,7 +320,7 @@ sub add_disk {
         my $dir = "/var/lib/libvirt/images";
         die "No file given" unless $args->{file};
         if ($args->{cdrom} or $args->{backingfile}) {
-            die "File $args->{file} not readable" unless -r $args->{file};
+            $self->run_cmd('ls ' . $args->{file}) && die "File $args->{file} does not exist on svirt host";
             $self->run_cmd(sprintf("rsync -av '$args->{file}' '${dir}/%s'", basename($args->{file}))) && die 'rsync failed';
```

Comment:
Why not just let rsync fail? I don't see any value on one more remote command

Reply:
yes, I'll just remove the line